src/UI/Component/Input/Field/Factory.php

```diff
@@ -348,5 +347,41 @@ public function password($label, $byline = null);
 	 * @return \ILIAS\UI\Component\Input\Field\Select
 	 */
 	public function select($label, array $options, $byline = null);
+
+
+	/**
+	 * ---
+	 * description:
+	 *   purpose: >
+	 *     A multi-select is used to allow users to pick several options from a list.
```

Comment:
Multi Select (there are many further down, I will not mark them all to not clutter the description too much.